README.md

```diff
@@ -26,7 +28,56 @@ Finally, make sure that [jsdom](https://github.com/jsdom/jsdom) 10.0.0 or later
 You can install it with
 
 ```bash
-$ npm install jsdom
+$ npm install jsdom --save-dev
+```
+
+Or with yarn if you prefer
+
+```bash
+$ yarn add jsdom --dev
 ```
 
 See [the Scaladoc](https://javadoc.io/doc/org.scala-js/scalajs-env-jsdom-nodejs_2.13/latest/org/scalajs/jsenv/jsdomnodejs/index.html) for other configuration options.
+
+## Usage - Writing a test
+
+To access the dom, you will need to install [`scala-js-dom`](https://github.com/scala-js/scala-js-dom):
+
+```scala
+libraryDependencies += "org.scala-js" %%% "scalajs-dom" % "2.1.0"
+```
+
+Create a test in your favourite test framework, here is an example using [munit](https://scalameta.org/munit/).
+
+```scala
+import org.scalajs.dom.document
+
+class ExampleJsDomTests extends munit.FunSuite {
+
+  test("Example jsdom test") {
+    val id      = "my fancy element"
+    val content = "Hi there and greetings!"
+
+    // Create a new div element
+    val newDiv = document.createElement("div");
```

Comment:
```suggestion
    val newDiv = document.createElement("div")
```
Various stray semicolons below as well.

Reply:
Oops! I started out with and MDN js example and modified...